packages/website/src/components/ShareDropdown.tsx

```diff
@@ -4,14 +4,21 @@ import { ShareIcon, FacebookIcon, TwitterIcon } from '../icons';
 
 type ShareOption = 'facebook' | 'twitter';
 
-const shareUrlMap: Record<ShareOption, string> = {
-  facebook: 'https://www.facebook.com/sharer/sharer.php?u=',
-  twitter: 'https://twitter.com/intent/tweet?url=',
+const shareUrlMap: Record<ShareOption, (any) => string> = {
+  facebook: ({ u }) => `https://www.facebook.com/sharer/sharer.php?t=${u}`,
+  twitter: ({ t }) => `https://twitter.com/intent/tweet?text=${t}`,
 };
```

Comment:
Can we clean these args up a little bit it just doesn't seem very clear. Maybe define a function for creating the share urls for facebook and twitter each with more clearly defined args.

I assume the changes to the twitter share no longer link to a specific URL since we aren't providing that query arg anymore. I would see if you can use both url and text for twitter and if that doesn't work just use the URL.

Reply:
Okay cool guess that works then